Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1
+FROM condaforge/mambaforge
+LABEL MAINTAINER="tardis.supernova.code@gmail.com"
+
+COPY conda-linux-64.lock /tmp
+RUN mamba create -n tardis --file /tmp/conda-linux-64.lock
+
+COPY . /tmp/repo
+RUN conda run -n tardis pip install /tmp/repo \
+    && echo "conda activate tardis" >> ~/.bashrc
+
+RUN mkdir /workdir
+COPY docs/quickstart.ipynb /workdir
+WORKDIR /workdir
+
+RUN conda clean --all \
+    && apt-get autoremove --purge -y \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
Just curious, why are we cleaning apt lists?

Reply:
From [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/):

> In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.